tests/live/test_userid.py

```diff
@@ -42,9 +44,60 @@ def test_06_unregister_str(self, fw, state_map):
         if not state.single_register:
             raise Exception("No single_register")
         ip, tag = state.single_register
-        fw.userid.unregister(ip, tag)
+        fw.userid.unregister(ip, (tag, "aaaaaaa"))
+        assert fw.userid.get_registered_ip() == {}
+
+    def test_07_error(self, fw):
+        fw.userid.clear_registered_ip()
+        ips = [testlib.random_ip() for x in range(10)]
+        tags = [testlib.random_name() for i in range(10)]
+        fw.userid.register(ips, tags)
+        fw.userid.unregister(ips, tags[9])
+        original = fw.userid.get_registered_ip()
+        new_tags = tags[0:9]
+        new_tags.append("aaaaaa")
+        fw.userid.unregister(ips[0], new_tags)
+        mod1 = fw.userid.get_registered_ip()
+        assert len(original.keys()) == len(mod1.keys()) + 1
+
+    def test_03_looper(self, fw, state_map):
```

Comment:
Looks like this is the function I wrote..?  Go ahead and remove it.  It has `print()`s in it, and there's no reason to have a loop and assert in a single test like this.  I think your other tests already do what this is trying, right..?